CatNote.API/Program.cs

```diff
@@ -21,7 +21,8 @@ public static void Main(string[] args)
 
         var connection = builder.Configuration.GetConnectionString("DefaultConnection");
         builder.Services.AddBusinessServices(connection);
-        builder.Services.AddMapperServices();
+
```

Comment:
also coud've done it like dat:
builder.Services.AddAutoMapper(AppDomain.CurrentDomain.GetAssemblies());

easier to maintain